cpp/ql/src/Likely Bugs/Arithmetic/SignedOverflowCheck.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+Testing for <code>signed</code> integer overflow by adding a
```

Comment:
```suggestion
Testing for a <code>signed</code> integer overflow by adding a
```

Reply:
Hmm... Adding `a` feels unnatural to me.  We may very well find _multiple_ occurrences of these overflow tests, so neither `a` nor `the` seem sensible.

Reply:
I'm with @zlaski-semmle on that one. In this context, _overflow_ behaves grammatically like _death_ or _malice_.

Reply:
Fair enough. You guys are closer to the audience than me.